app/views/posts/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title, "#{@user.name} | #{@post.title}" %>
+<div class="mainContainer">
+<%= render "user_post", post: @post%>
+<p> <%= @user.posts_counters%> </p>
+<div class="commentsDiv">
+  <% @comments.each do |comment| %>
+    <div >
+      <p>
+        <span><strong><%= comment.user.name %>:</strong></span>
+        <span><%= comment.text %></span>
+      </p>
+    </div>
+  <% end %>
+  </div>
+  <%= form_with model: @comment, url: user_post_comments_path(post_id: @post.id), method: :post, scope: :comment do |f| %>
+    <div>
+      <%= f.text_area :text, placeholder: "Add Comment" %>
+      <%= f.submit "Comment" %>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Great work displaying the comments. There is one more feature to add. The User should be able to delete their comments on a post if they are the ones that created it. You created an action in the `Posts` controller for handling it but it was not implemented in the UI. Kindly implement this change as it is a major requirement. 😎